USFMToolsSharp.Renderers.USX/USXRenderer.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using USFMToolsSharp.Models.Markers;
+
+namespace USFMToolsSharp.Renderers.USX
+{
+    public class USXRenderer
+    {
+        public List<string> UnrenderableTags;
+        public USXConfig ConfigurationUSX;
+        private IList<string> TOCEntries;
+        
+        public USXRenderer()
+        {
+            UnrenderableTags = new List<string>();
+            ConfigurationUSX = new USXConfig();
+            TOCEntries = new List<string>();
+        }
+
+        public USXRenderer(USXConfig config) : this()
+        {
+            ConfigurationUSX = config;
+        }
+        public string Render(USFMDocument input)
+        {
+            var encoding = GetEncoding(input);
+            var output = new StringBuilder();
+            var bodyContent = new StringBuilder();
+
+            if (!ConfigurationUSX.partialUSX)
+            {
+                bodyContent.AppendLine($"<?xml version=\"1.0\" encoding=\"{encoding}\">");
+                bodyContent.AppendLine("<usx version=\"3.0\">");
+            }
+
+            foreach (Marker marker in input.Contents)
```

Comment:
You aren't going to need to do this body contents, you can just write straight to output. The reason this is is split in this case is that in the HTML renderer needed to split the footnote caller and actual footnotes. You aren't going to need to do that.